### PR TITLE
Added contributing.md and security.md files

### PR DESCRIPTION
**Pull request by MohammedAman23**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+-   documentation changes,
+-   code changes,
+-   keeping up the main branch.
+
+## General information
+
+For specific proposals, please provide them as [pull requests](https://github.com/datakaveri/iudx-aaa-server/pulls) or [issues](https://github.com/datakaveri/iudx-aaa-server/issues) via our [GitHub site](https://github.com/datakaveri/iudx-aaa-server).
+
+
+The [READMEmd](https://github.com/datakaveri/iudx-aaa-server/blob/main/README.md) file explains how to install the program locally (highly recommended if you're going to make code changes). It also provides a quick start guide.
```
Comment (ThorodanBrom) [inline CONTRIBUTING.md:21]: Put dot here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Pull requests are preferred, since they are specific. For more about how to create a pull request, see
+<https://help.github.com/articles/using-pull-requests/>.
+
+We recommend creating different branches for different (logical) changes, and creating a pull request when you're done into the main branch. See the GitHub documentation on [creating branches](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/) and [using pull requests](https://help.github.com/articles/using-pull-requests/).
+
+### How we handle proposals
+
+We use GitHub to track proposed changes via its [issue tracker](https://github.com/coreinfrastructure/best-practices-badge/issues) and [pull requests](https://github.com/coreinfrastructure/best-practices-badge/pulls).
+Specific changes are proposed using those mechanisms.
```
Comment (ThorodanBrom) [inline CONTRIBUTING.md:34]: Update URLs here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+If you find a significant vulnerability, or evidence of one, please send an email to the security contacts that you have such information, and we'll tell you the next steps.
+For now, the security contacts are:
+IUDX admin <[admin@iudx.org.in](mailto:support@iudx.org.in)> and 
+IUDX Support <[support@iudx.org.in](mailto:support@iudx.org.in)>.
+
+Please use an email system (like Gmail) that supports hop-to-hop encryption using STARTTLS when reporting vulnerabilities. Examples of such systems include Gmail, Outlook.com, and runbox.com. 
+See [STARTTLS Everywhere](https://starttls-everywhere.org/) if you wish to learn more about efforts to encourage the use of STARTTLS.
+Your email client should use encryption to communicate with your email system (i.e., if you use a web-based email client then use HTTPS, and if you use email client software then configure it to use encryption). Hop-to-hop encryption isn't as strong as end-to-end encryption,
+but we've decided that it's strong enough for this purpose and it's much easier to get everyone to use it.
```
Comment (ThorodanBrom) [inline CONTRIBUTING.md:70]: Don't know if this is necessary

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+# Contributing
+
+<!-- SPDX-License-Identifier: (MIT OR CC-BY-3.0+) -->
+
+Feedback and contributions are very welcome!
+
+Here's help on how to make contributions, divided into the following sections:
+
+-   general information,
+-   [vulnerability reporting](https://github.com/coreinfrastructure/best-practices-badge/blob/main/CONTRIBUTING.md#how_to_report_vulnerabilities),
```
Comment (ThorodanBrom) [inline CONTRIBUTING.md:11]: Update URL here

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+We encourage tests to be created first, run to ensure they fail, and then add code to implement the test (aka test driven development).
+
+
+### Other tools
+
+Here are some other tools we sometimes use for checking quality or security.
+
+* OWASP ZAP web application security scanner.
+  You are encouraged to use this and other web application scanners to find and
+  fix problems.
```
Comment (abhi4578) [inline CONTRIBUTING.md:106]: We can mention JaCoCo code coverage tool, postman and newman.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
```
Comment (abhi4578) [inline CONTRIBUTING.md:107]: Add this section
### Testing during continuous integration

Note that we also use
[Jenkins](https://jenkins.iudx.io/)
for continuous integration tools to check changes
after they are checked into GitHub; if they find problems, please fix them.
These run essentially the Junit tests, JaCoCo code coverage tests, security tests using OWASP ZAP and integration tests using postman and newman

**Review by abhi4578 — CHANGES_REQUESTED** [review]

**Comment by abhi4578** [comment]
retest this please

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**